core/src/components/input/input.scss

```diff
@@ -384,6 +377,16 @@
 
   align-items: center;
 
+  /**
+   * Label text should not extend
+   * beyond the bounds of the input.
+   * However, we do not set the max
+   * width to 100% because then
+   * only the label would show and users
+   * would not be able to see what they are typing.
+   */
+  max-width: 200px;
```

Comment:
Does native do this? I feel like this should be an implementation detail; if the app has a really long label, that's on the dev. This feels like `fixed` label behavior sneaking in where it might not be wanted.

Reply:
Aha, works for me 👍